### ISSUE TITLE
Unable to obtain build result from private repository using PAT

### ISSUE THREAD

**Issue by grmmvv** [lemurheavy/coveralls-public#1463]
Hi!

My problem looks the same as #1166 and #1219 

I'm trying to get build result in JSON format, so I've created PAT for my account [here](https://coveralls.io/account) and using it in my GET request to `https://coveralls.io/builds/${PRIVATE_REPO_BUILD_ID}.json` but it always return 403 Forbidden.

Is there any ideas how I can fix it?

**Comment by tawhidm** [comment]
You need to pass the repo token for private repos: 
`https://coveralls.io/builds/${PRIVATE_REPO_BUILD_ID}.json?repo_token=${token}`

**Comment by grmmvv** [comment]
It helps! Thank you!

**Comment by afinetooth** [comment]
BTW, in case you do need to use that PAT (personal access token), as for requesting repo data from coveralls, here's how to get it and use it:

You'll can generate a PAT from your account page, here:
https://coveralls.io/account

Once you have the value, you'll want to send it in the following format in the HEADER of your request, specifically in the **Authorization** header, where the value should be given as follows:

```
Authorization: "token eOHdLJGkyIOirzbKfIJUau0D2CRPXLJCDfqzR"
```

Here's how it would look in `curl`, if you were, for instance, requesting info on your repo:

```
curl --location --request GET 'https://coveralls.io/api/repos/github/afinetooth/coveralls-demo-ruby' \
--header 'Accept: application/json' \
--header 'Content-Type: application/json' \
--header 'Authorization: token eOHdLJGkyIOirzbKfIJUau0D2CRPXLJCDfqzR'
```

**Comment by nychang** [comment]
@grmmvv It sounds like you were able to get the json. 🎉 

Where do I find this mysterious `repo_token`? 🤔  The [docs](https://docs.coveralls.io/api-reference) say:

> `repo_token` The secret token for your repository, found at the bottom of your repository’s page on Coveralls.

But I don't see a token anywhere on the page.

I tried using the following but both options still give me 403s:

- personal api token generated from coveralls.io/account
- personal access token with all permissions generated from github.com/settings/tokens

I am using Postman with:
```
Accept: application/json
Content-Type: application/json
Authorization: token <Coveralls personal api token>
Method: GET
Url: https://coveralls.io/github/github_username/repo_name.json?repo_token=<TOKEN>
```

Where `<TOKEN>` is the PAT I got from either Coveralls or GitHub, both of which still give 403s.

Thank you for your help! 🙌
